README.md

```diff
@@ -38,11 +49,15 @@ To gain a better understanding of the underlying API calls the SDK makes, feel f
 
 ## Internal implementation details
 
-This project is written in [TypeScript](https://www.typescriptlang.org/) and built with [tsdx](https://github.com/formium/tsdx), thus CJS, ESModules, and UMD module formats are supported.
+This project is written in [TypeScript](https://www.typescriptlang.org/) and built with [tsdx](https://github.com/formium/tsdx).
+
+CJS, ESM, and UMD module builds are provided.
 
 The http client used by the sdk is [got](https://github.com/sindresorhus/got).
 
-All resource operations are mounted as sub-APIs on a `Clerk` object and return promises that either resolve with their expected resource types or reject with the error types described below.
+All resource operations are mounted as sub-APIs on a `Clerk` class and return promises that either resolve with their expected resource types or reject with the error types described below.
```

Comment:
👍 Do we coin the term `sub-APIs`? I like it.

Reply:
Well, it is kind of deceiving.
It's not like we have a main api and sub-apis.
Trying to refer to a grouping of api operations for a particular resource.
Naming suggestions most welcome.